assets/src/apps/delivery/layouts/deck/DeckLayoutView.tsx

```diff
@@ -42,13 +43,40 @@ const InjectedStyles: React.FC<{ css?: string }> = (props) => {
   return injected ? <style>{injected}</style> : null;
 };
 
+const getActualOwnerDetails = (expression: string, sequence: any) => {
+  const [activityId, part] = expression.split('|');
+  const activitySequence = sequence.find((entry: any) => entry?.custom?.sequenceId === activityId);
+  const activityLayerId = activitySequence?.custom?.layerRef;
+  let expressionValue = getValue(expression, defaultGlobalEnv);
```

Comment:
should pass in `env = defaultGlobalEnv` as a parameter instead of requiring it here, or possibly without a default, makes it more flexible & testable